package.json

```diff
@@ -40,6 +42,7 @@
     "@types/lodash": "^4.14.178",
     "@types/moment": "^2.13.0",
     "@types/react-native": "^0.66.15",
+    "@types/react-native-popup-dialog": "^0.16.4",
```

Comment:
same as abov

Reply:
regarding this i get an error when i remove it i get
Could not find a declaration file for module 'react-native-popup-dialog'.
Try `npm i --save-dev @types/react-native-popup-dialog`, so i shpuld use only this command without yarn add react-native-popup-dialog ?